### PR TITLE
MapExists boolean dashboard update

### PR DESCRIPTION
**Pull request by djnunez-aot**
-Move Grid container inside project location to allow Project location to be rendered at all times

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the met-public license (Apache 2.0).

### CODE REVIEW

File: met-web/src/components/dashboard/EngagementsAccordion.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ const EngagementsAccordion = ({
     const [openedEngagements, setOpenedEngagements] = useState<number[]>([]);
     const isMobile = useMediaQuery((theme: Theme) => theme.breakpoints.down('xs'));
     const [projectMapData, setProjectMapData] = React.useState<Map | null>(null);
-    const mapExists = projectMapData?.latitude && projectMapData?.longitude;
+    const mapExists = (projectMapData?.latitude && projectMapData?.longitude) ?? true;
```
Comment (jadmsaadaot) [inline met-web/src/components/dashboard/EngagementsAccordion.tsx:29]: is this the result we're looking for here:
image

**Comment by jadmsaadaot** [comment]
please fix description, and noticed code smells

File: met-web/src/components/dashboard/EngagementsAccordion.tsx (hunk 1 above)
Comment (djnunez-aot) [reply met-web/src/components/dashboard/EngagementsAccordion.tsx:29]: no at all

File: met-web/src/components/publicDashboard/KPI/ProjectLocation.tsx [hunk 2]
```diff
@@ -14,10 +14,16 @@ import { HTTP_STATUS_CODES } from 'constants/httpResponseCodes';
...
 
-const ProjectLocation = ({ engagement, engagementIsLoading, handleProjectMapData }: SurveysCompletedProps) => {
+const ProjectLocation = ({
+    engagement,
+    engagementIsLoading,
+    updateLoading,
+    handleProjectMapData,
+}: SurveysCompletedProps) => {
     const [data, setData] = useState<Map | null>(null);
     const [isLoading, setIsLoading] = useState(true);
```
[the remark this answers is not in the document]
Comment (djnunez-aot) [reply met-web/src/components/publicDashboard/KPI/ProjectLocation.tsx:28]: we can leverage it but I think we may need to use the <When> component still to change the layout when it is present or not or i can nest the grid inside <ProjectLocation> instead

**Review by jadmsaadaot — APPROVED** [review]

**Pull request merged**